source/dealii/dealii_matrix_free_operator.cc

```diff
@@ -261,15 +266,15 @@ DealIIMatrixFreeOperator<VectorType>::build_range_vector() const
 template <typename VectorType>
 size_t DealIIMatrixFreeOperator<VectorType>::grid_complexity() const
 {
-  ASSERT_THROW_NOT_IMPLEMENTED();
+  //  ASSERT_THROW_NOT_IMPLEMENTED();
 
   return typename DealIIMatrixFreeOperator<VectorType>::value_type{};
 }
 
 template <typename VectorType>
 size_t DealIIMatrixFreeOperator<VectorType>::operator_complexity() const
 {
-  ASSERT_THROW_NOT_IMPLEMENTED();
+  //  ASSERT_THROW_NOT_IMPLEMENTED();
```

Comment:
Just remove the two `ASSERT`

Reply:
It was meant to remember the return is garbage.  Want me to comment instead?

@Rombur says yes please!

Reply:
Done